sdcm/send_email.py

```diff
@@ -77,6 +77,7 @@ def __del__(self):
 
 class BaseEmailReporter():
 
+    _attachments_size_limit = 20971520
     fields = []
```

Comment:
No need to put a underscore, i.e. this is not really private.

And since it's a constant, can be in upper case.